layouts/partials/scaler-layout.html

```diff
@@ -0,0 +1,58 @@
+<br />
```

Comment:
Do we need these?

Think we can drop those:
![image](https://user-images.githubusercontent.com/4345663/185042323-79bf0d41-ed18-4994-aa5b-dacbe05afcab.png)

Reply:
yeah...we don't. But I think it's best I fix it in a separate PR since there is an open issue for it already

Reply:
Ok, I forgot about the issue, thanks!